loki/transformations/tests/test_data_offload.py

```diff
@@ -42,10 +43,45 @@ def fixture_config():
         },
     }
 
+@pytest.fixture(name="parkind_mod")
+def fixture_parkind_mod(tmp_path, frontend):
+    fcode = """
+    module parkind1
+      integer, parameter :: jprb=4
+    end module
+    """
+    return Module.from_source(fcode, frontend=frontend, xmods=[tmp_path])
+
+@pytest.fixture(name="field_module")
+def fixture_field_module(tmp_path, frontend):
```

Comment:
Neat! 😏

Reply:
Yeah! It was a good suggestion to use a fixture for this to reduce boilerplate in the tests.